custom_model_runner/datarobot_drum/drum/push.py

```diff
@@ -29,7 +29,7 @@ def _convert_target_type(unconverted_target_type):
     elif unconverted_target_type == TargetType.BINARY.value:
         return dr_client.TARGET_TYPE.BINARY
     elif unconverted_target_type == TargetType.ANOMALY.value:
-        return dr_client.CUSTOM_MODEL_TARGET_TYPE.ANOMALY
+        return dr_client.enums.CUSTOM_MODEL_TARGET_TYPE.ANOMALY
```

Comment:
Could we unify enums being used in this block?
`dr_client.TARGET_TYPE` vs `dr_client.enums.CUSTOM_MODEL_TARGET_TYPE`

Reply:
@jlpolit there was a reason why no. Anomaly doesn't show up in the normal modeling target types.

Reply:
yeah I think for now we need to check the custom model enums